docs/man/docker.1.md

```diff
@@ -90,7 +90,7 @@ unix://[/path/to/socket] to use.
   Set key=value labels to the daemon (displayed in `docker info`)
 
 **--log-driver**="*json-file*|*syslog*|*none*"
```

Comment:
What is the rational for showing other options?  Why not just do this like the cli.md:

 --log-driver="json-file"               Default driver for container logs